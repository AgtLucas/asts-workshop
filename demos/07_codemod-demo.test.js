import * as babel from 'babel-core'
import jQueryHidePlugin from './07_codemod-demo'

test('codemods imports of CommonJS modules', () => {
  const source = `
    $(el).hide()
    foo.hide()
  `
  const {code} = babel.transform(source, {
    babelrc: false,
    plugins: [jQueryHidePlugin],
  })
  expect(code).toMatchSnapshot()
})
