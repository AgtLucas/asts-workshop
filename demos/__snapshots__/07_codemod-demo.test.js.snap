// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`codemods imports of CommonJS modules 1`] = `
"
el.style.display = 'hide';
foo.hide();"
`;
